src/pages/offer-details/OfferDetails.tsx

```diff
@@ -60,6 +66,14 @@ const OfferDetails = () => {
           onBookmarkClick={onBookmarkClick}
         />
       </AppCard>
+      <AppCard sx={styles.wrapper}>
+        <ComentsBlock
```

Comment:
you should add title as a props, because now we only have what students say, but we will use comment for students too

Reply:
Fixed